app/models/condition.rb

```diff
@@ -0,0 +1,7 @@
+class Condition < ApplicationRecord
+  has_paper_trail
+
+  belongs_to :routing_page, class_name: "Page"
+  belongs_to :check_page, class_name: "Page", optional: true
+  belongs_to :goto_page, class_name: "Page", optional: true
```

Comment:
he idea here is that for a Condition object, you can call c.routing_page to find out which page it's attached to, c.check_page to find the page that it's checking, and c.goto_page to find the page that it skips to

Reply:
@SamJamCul These differ slightly from the [equivalent names in the ADR](https://github.com/alphagov/forms/pull/65/files#diff-8bcea9845a999e062259cd5f1a83bb9412e41bb0a60b6d937eab0d717d8c06adR15-R18), so we should make sure we document these somewhere

Reply:
I don't know if we need to be that tethered to what's in the ADR. The ADR is meant to capture the decision to build the routing work as a table in our DB; the way we implement it exactly isn't something we can have predicted in the ADR. I think this PR could be linked as part of the ADR somewhere, to show that this work was following on from it?

Reply:
Yeah that makes sense I think 👍